examples/typescript/multi_agent_transfer.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ */
+
+import { Account, AccountAddress, Aptos, AptosConfig, Network, StructTag, TypeTagStruct, U64 } from "aptos";
+
+// TODO: There currently isn't a way to use the APTOS_COIN in the COIN_STORE due to a regex
+const APTOS_COIN = "0x1::aptos_coin::AptosCoin";
+const COIN_STORE = `0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>`;
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 100_000_000;
+const TRANSFER_AMOUNT = 10;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ */
+const balance = async (aptos: Aptos, name: string, address: AccountAddress) => {
+  let resource = await aptos.getAccountResource({ accountAddress: address.toUint8Array(), resourceType: COIN_STORE });
+  let amount = Number((resource.data as { coin: { value: string } }).coin.value);
+
+  console.log(`${name}'s balance is: ${amount}`);
+  return amount;
+};
+
+const example = async () => {
+  console.log(
+    "This example will create two accounts (Alice and Bob), fund them, create an object, and transfer the object between them using move scripts and a multi-agent transaction.",
+  );
+
+  // Setup the client
+  const config = new AptosConfig({ network: Network.DEVNET });
```

Comment:
Should be `new AptosConfig();` obj as param is removed.

And devnet is the default now.